src/ui/qml/GeneralPreferences.qml

```diff
@@ -24,6 +24,31 @@ ColumnLayout {
         }
     }
 
+    CheckBox {
+        text: qsTr("Play audio notifications")
+        checked: uiSettings.data.playAudioNotification || false
+        onCheckedChanged: {
+            uiSettings.write("playAudioNotification", checked)
+        }
+    }
+    RowLayout {
+        Label { text: qsTr("volume") }
```

Comment:
Use title case. This would look good if it was indented slightly on the left, so it's obvious that it is part of "Play audio notifications"